R/packrat-mode.R

```diff
@@ -182,6 +182,10 @@ afterPackratModeOn <- function(project,
     options(repos = repos)
   }
 
+  # Set a secure download method if any of the repos URLs use https
```

Comment:
Is this something we should do every time, or only if we actually set the repos in line 182?